### ISSUE TITLE
Invocation of `Compilation#buildModule()` is wrong for Webpack 2

### ISSUE THREAD

**Issue by bryanforbes** [dojo/cli-build#68]
The API for `Compilation#buildModule()` changed in Webpack 2 and `InjectModulesPlugin` was not updated to accommodate this change. Currently, the following is done:

```ts
compilation.buildModule(module, (error?: Error) => {
```

But it should be called like so:

```ts
compilation.buildModule(module, false, null, null, (error?: Error) => {
```

I discovered this while writing typings for Webpack for #53.

**Comment by bryanforbes** [comment]
I will submit a fix for this issue as part of the pull request to fix #53.

**Comment by mwistrand** [comment]
This was resolved by #69.
